omp/matrix/dense_kernels.cpp

```diff
@@ -202,7 +209,33 @@ GKO_INSTANTIATE_FOR_EACH_VALUE_TYPE(GKO_DECLARE_DENSE_COMPUTE_NORM2_KERNEL);
 template <typename ValueType, typename IndexType>
 void convert_to_coo(std::shared_ptr<const OmpExecutor> exec,
                     matrix::Coo<ValueType, IndexType> *result,
-                    const matrix::Dense<ValueType> *source) NOT_IMPLEMENTED;
+                    const matrix::Dense<ValueType> *source)
+{
+    auto num_rows = result->get_size()[0];
+    auto num_cols = result->get_size()[1];
+    auto num_nonzeros = result->get_num_stored_elements();
+
+    auto row_idxs = result->get_row_idxs();
+    auto col_idxs = result->get_col_idxs();
+    auto values = result->get_values();
+
+    auto idxs = 0;
```

Comment:
The change (`auto -> size_type`) is lost

Reply:
Thanks @yhmtsai , not sure how I messed that up. even though it does not belong there, I will add this change in the adaptive Jacobi OMP kernel merge.